Adding information on how to start the trace on a read scale-out replica in Azure SQL Managed Instance (#9693)
MicrosoftDocs/sql-docs: azure-sql/database/read-scale-out.md

### Monitor read-only replicas with Extended Events

An extended event session can't be created when connected to a read-only replica. However, in Azure SQL Database and Azure SQL Managed Instance, the definitions of database-scoped [Extended Event](xevent-db-diff-from-svr.md) sessions created and altered on the primary replica replicate to read-only replicas, including geo-replicas, and capture events on read-only replicas.

> In Azure SQL Managed Instance, to start a trace on a read-only replica, you must first start the trace on the primary replica before you can start the trace on the read-only replica. If you do not first start the trace on the primary replica, you will receive the following error when attempting to start the trace on the read-only replica:

Reviewer: ```suggestion
In Azure SQL Database, an extended event session on a read-only replica that is based on a session definition from the primary replica can be started and stopped independently of the session on the primary replica. If you don't first start the trace on the primary replica, you receive the following error when attempting to start the trace on the read-only replica:
```

> In Azure SQL Database, an extended event session on a read-only replica that is based on a session definition from the primary replica can be started and stopped independently of the session on the primary replica. If you don't first start the trace on the primary replica, you receive the following error when attempting to start the trace on the read-only replica:

Reviewer: ```suggestion
In Azure SQL Database, an extended event session on a read-only replica that is based on a session definition from the primary replica can be started and stopped independently of the session on the primary replica.
 
In Azure SQL Managed Instance, to start a trace on a read-only replica, you must first start the trace on the primary replica before you can start the trace on the read-only replica. If you do not first start the trace on the primary replica, you will receive the following error when attempting to start the trace on the read-only replica:
```

### Monitor read-only replicas with Extended Events

An extended event session can't be created when connected to a read-only replica. However, in Azure SQL Database and Azure SQL Managed Instance, the definitions of database-scoped [Extended Event](xevent-db-diff-from-svr.md) sessions created and altered on the primary replica replicate to read-only replicas, including geo-replicas, and capture events on read-only replicas.

In Azure SQL Database, an extended event session on a read-only replica that is based on a session definition from the primary replica can be started and stopped independently of the session on the primary replica.
 

> In Azure SQL Managed Instance, to start a trace on a read-only replica, you must first start the trace on the primary replica before you can start the trace on the read-only replica. If you do not first start the trace on the primary replica, you will receive the following error when attempting to start the trace on the read-only replica:

Reviewer: ```suggestion
In Azure SQL Managed Instance, to start a trace on a read-only replica, you must first start the trace on the primary replica before you can start the trace on the read-only replica. If you don't first start the trace on the primary replica, you receive the following error when attempting to start the trace on the read-only replica:
```
